spring-boot/line-bot-spring-boot-client/src/main/java/com/linecorp/bot/spring/boot/core/configuration/LineBotAutoConfiguration.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.bot.spring.boot.core.configuration;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.linecorp.bot.audience.client.ManageAudienceBlobClient;
+import com.linecorp.bot.audience.client.ManageAudienceClient;
+import com.linecorp.bot.client.base.channel.ChannelTokenSupplier;
+import com.linecorp.bot.client.base.channel.FixedChannelTokenSupplier;
+import com.linecorp.bot.liff.client.LiffClient;
+import com.linecorp.bot.messaging.client.MessagingApiBlobClient;
+import com.linecorp.bot.messaging.client.MessagingApiClient;
+import com.linecorp.bot.spring.boot.core.properties.LineBotProperties;
+
+/**
+ * Also refers for web only beans definition.
+ */
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(LineBotProperties.class)
+public class LineBotAutoConfiguration {
+    private final LineBotProperties lineBotProperties;
+
+    public LineBotAutoConfiguration(LineBotProperties lineBotProperties) {
+        this.lineBotProperties = lineBotProperties;
+    }
+
+    /**
+     * Expose {@link FixedChannelTokenSupplier} as {@link Bean}
+     * in case of no other definition for {@link ChannelTokenSupplier} type.
+     */
+    @Bean
+    @ConditionalOnMissingBean(ChannelTokenSupplier.class)
+    public ChannelTokenSupplier channelTokenSupplier() {
+        final String channelToken = lineBotProperties.channelToken();
+        return FixedChannelTokenSupplier.of(channelToken);
+    }
+
+    /**
+     * Expose {@link MessagingApiClient} as {@link Bean}.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public MessagingApiClient messagingApiClient(
```

Comment:
I forgot `manager.line.biz` client in https://github.com/line/line-bot-sdk-java/pull/956. Do you have any plan to support it?

Reply:
Oops. I forgot it.  I'll implement it in next PR.

Reply:
Thank you!